public/style.css

```diff
@@ -0,0 +1,3 @@
+.ql-container.ql-snow {
+  border: none!important;
+}
```

Comment:
You'll need to add a newline here @samuelbwambale.

Reply:
Thanks for the feedback @RonKbS